config/initializers/geocoder.rb

```diff
@@ -28,16 +28,60 @@
 if Rails.env.test?
```

Comment:
As these are tests, I think they should be in a file in `spec/support` ?

Reply:
:+1: have pushed an update moving them to spec/support